File: createJson/src/main/java/com/ruzhan/jsonfile/data/Marvel.java
package com.ruzhan.jsonfile.data;

import com.ruzhan.jsonfile.model.Introduce;
import com.ruzhan.jsonfile.model.Movie;
import com.ruzhan.jsonfile.model.MovieDetail;
import com.ruzhan.jsonfile.model.Video;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by ruzhan123 on 2018/7/3.
 */
public class Marvel {

    private static final int ID = 1903;
    private static final String PLAY_COUNT = "1";

    private static final String TITLE = "Marvel 75 Years: From Pulp to Pop";
    private static final String TAG = "电影·科幻·纪录片";

    private static final String IMAGE = "https://raw.githubusercontent.com/ruzhan123/Lion/master/json/api/image/marve.jpg";

    private static final String DESC = "庆祝漫威75周年，回顾漫威这75年的历史，并且展望未来的发展。";

    public static final Movie movie = new Movie();
    public static final MovieDetail movieDetail = new MovieDetail();

    private static final List<Introduce> introduceList = new ArrayList<>();
    private static final List<Video> videoList = new ArrayList<>();

    static {
        introduceList.add(new Introduce(Introduce.TEXT,
                DESC, ""));

        introduceList.add(new Introduce(Introduce.IMAGE, "",
                "https://raw.githubusercontent.com/ruzhan123/Lion/master/json/api/image/marve01.jpg"));

        introduceList.add(new Introduce(Introduce.TEXT,
                "节目将由《复仇》女主，也就是《美国队长：冬日战士》特工13/Sharon扮演者——艾米丽·万凯普（Emily VanCamp）主持。", ""));


        videoList.add(new Video(String.valueOf(ID + 11),
                "揭秘漫威的成功之路", IMAGE, PLAY_COUNT, 1, "电影",
                "http://player.bilibili.com/player.html?aid=10485121&cid=17314449&page=1"));

        movie.id = String.valueOf(ID);
        movie.title = TITLE;
        movie.tag = TAG;
        movie.desc = DESC;
        movie.image = IMAGE;

        movieDetail.id = String.valueOf(ID + 3);
        movieDetail.movieId = String.valueOf(ID);
        movieDetail.title = TITLE;
        movieDetail.tag = TAG;
        movieDetail.desc = DESC;
        movieDetail.image = IMAGE;

        movieDetail.introduces = new ArrayList<>();
        movieDetail.introduces.addAll(introduceList);

        movieDetail.videos = new ArrayList<>();
        movieDetail.videos.addAll(videoList);
    }
}

File: createJson/src/main/java/com/ruzhan/jsonfile/data/Marvel.kt
package com.ruzhan.jsonfile.data

import com.ruzhan.jsonfile.model.Introduce
import com.ruzhan.jsonfile.model.Movie
import com.ruzhan.jsonfile.model.MovieDetail
import com.ruzhan.jsonfile.model.Video

import java.util.ArrayList

/**
 * Created by ruzhan123 on 2018/7/3.
 */
object Marvel {

    private val ID = 1903
    private val PLAY_COUNT = "1"

    private val TITLE = "Marvel 75 Years: From Pulp to Pop"
    private val TAG = "电影·科幻·纪录片"

    private val IMAGE = "https://raw.githubusercontent.com/ruzhan123/Lion/master/json/api/image/marve.jpg"

    private val DESC = "庆祝漫威75周年，回顾漫威这75年的历史，并且展望未来的发展。"

    val movie = Movie()
    val movieDetail = MovieDetail()

    private val introduceList = ArrayList<Introduce>()
    private val videoList = ArrayList<Video>()

    init {
        introduceList.add(Introduce(Introduce.TEXT,
                DESC, ""))

        introduceList.add(Introduce(Introduce.IMAGE, "",
                "https://raw.githubusercontent.com/ruzhan123/Lion/master/json/api/image/marve01.jpg"))

        introduceList.add(Introduce(Introduce.TEXT,
                "节目将由《复仇》女主，也就是《美国队长：冬日战士》特工13/Sharon扮演者——艾米丽·万凯普（Emily VanCamp）主持。", ""))


        videoList.add(Video((ID + 11).toString(),
                "揭秘漫威的成功之路", IMAGE, PLAY_COUNT, 1, "电影",
                "http://player.bilibili.com/player.html?aid=10485121&cid=17314449&page=1"))

        movie.id = ID.toString()
        movie.title = TITLE
        movie.tag = TAG
        movie.desc = DESC
        movie.image = IMAGE

        movieDetail.id = (ID + 3).toString()
        movieDetail.movieId = ID.toString()
        movieDetail.title = TITLE
        movieDetail.tag = TAG
        movieDetail.desc = DESC
        movieDetail.image = IMAGE

        movieDetail.introduces = ArrayList()
        movieDetail.introduces.addAll(introduceList)

        movieDetail.videos = ArrayList()
        movieDetail.videos.addAll(videoList)
    }
}
